validator/src/main/java/com/amazon/aoc/helpers/RetryHelper.java

```diff
@@ -37,11 +37,16 @@ public static void retry(
       throws Exception {
     while (retryCount-- > 0) {
       try {
-        log.info("still can retry for {} times", retryCount);
+        log.info("retry attempt left : {} ", retryCount);
         retryable.execute();
         return;
       } catch (Exception ex) {
-        log.error("exception during retry, you may ignore it", ex);
+        log.info("retrying after 10 seconds");
```

Comment:
i remember the interval is a parameter, not a fixed 10 seconds

Reply:
sure will add param here.